### PR TITLE
fix: workspace duplicate name handling causes unhandled 500 errors in UI

### PR DESCRIPTION
**Pull request by akozhuharov**
How to reproduce:

1. Create a workspace with a name - for example test-iac, save it
2. Create another workspace with the same name 
3. 
Screenshot 2024-06-25 at 14 24 30

### CODE REVIEW

File: ui/src/domain/Workspaces/Create.jsx
Comment (alfespa17) [inline ui/src/domain/Workspaces/Create.jsx:1]: This part looks good

File: api/src/main/java/org/terrakube/api/rs/workspace/Workspace.java
[the remark this answers is not in the document]
Comment (akozhuharov) [reply api/src/main/java/org/terrakube/api/rs/workspace/Workspace.java:1]: that's fair, then I was thinking something along the lines of a 400 response and handling it in the UI, how does that sound ?

[the remark this answers is not in the document]
Comment (alfespa17) [reply api/src/main/java/org/terrakube/api/rs/workspace/Workspace.java:1]: Yes adding some message in the UI sound good to me

[the remark this answers is not in the document]
Comment (akozhuharov) [reply api/src/main/java/org/terrakube/api/rs/workspace/Workspace.java:1]: Removed this, will think of a better way

**Pull request merged**